app.js

```diff
@@ -62,14 +62,14 @@ app.use(require('./controllers'));
 
 // If no route has matched, return 404
 app.use((req, res) => {
-  res.status(404).render('404.hbs');
+  res.status(404).render('404.hbs', {nonce1: req.nonce1, nonce2: req.nonce2});
 });
 
 // Basic error handler
-app.use((err, req, res) => {
+app.use((err, req, res, _) => {
```

Comment:
I need the 4th variable to get this code executed, I was using next but that gave lint errors, it works with underscore.